src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/reporting/ndmis/performance/ComplexityProcessor.kt

```diff
@@ -17,8 +17,7 @@ class ComplexityProcessor() : SentReferralProcessor<List<ComplexityData>> {
         interventionTitle = referral.intervention.title,
         serviceCategoryId = it.id,
         serviceCategoryName = it.name,
-        complexityLevelTitle = it.complexityLevels.find {
-          complexityLevel ->
+        complexityLevelTitle = it.complexityLevels.find { complexityLevel ->
```

Comment:
this changes the logic in the line below... 

`it` is the selected service category